core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -40,6 +41,17 @@ oppia.directive('simpleEditorSidebar', [
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.sidebarModeService = SimpleEditorSidebarModeService;
+          $scope.addNewQuestion = function() {
+            if (!SimpleEditorManagerService.canAddNewQuestion()) {
+              return;
+            }
+            $scope.sidebarModeService.setModeToReadonly();
```

Comment:
I was just wondering, Whether we should leave it to the user to switch the mode. changing the mode sometimes floods the sidebar with expanded questions and this UI change (from all questions collapsed to all expanded) is not so graceful. I personally don't like so abrupt change. It's just my openion and the experience may differ from person to person. What do you feel about it @saeedjassani  and @seanlip  ?